docs/orchestration/global-hooks/guides.mdx

```diff
@@ -0,0 +1,47 @@
+---
+title: Global Hooks Guide
+sidebarTitle: "Guide"
+---
+<img
+  alt="Penguin Fishing"
+  src="https://github.com/mage-ai/assets/blob/main/penguin_fishing.gif?raw=True"
+/>
+Follow this step by step guide on creating a global hook in Mage.
```

Comment:
Let's move this above the gif.

Reply:
Moved above, see attached pic from Mintlify dev.
<img width="1213" alt="Screenshot 2024-06-19 at 9 12 45 AM" src="https://github.com/mage-ai/mage-ai/assets/92221031/0119ffd8-48dc-4843-be9e-977e9846e7dc">

Reply:
@colefreeman Actually, can you please add it as a `description` in the header? So it appears directly under the title. Spacing looks kinda off here. You can look at many other doc mdx files for an example of the syntax, like this file: `docs/orchestration/global-data-products/overview.mdx`.

Looks something like:
```
---
title: "Global data products"
sidebarTitle: "Global data products"
description: "..."
---
```

Reply:
Same goes for the other file

Reply:
Moved to description tested in mintlify dev
<img width="609" alt="Screenshot 2024-06-19 at 2 34 39 PM" src="https://github.com/mage-ai/mage-ai/assets/92221031/928239b0-436a-485b-86bd-7c5f12205a02">